Sources/BraveNews/Customize/ChannelLabel.swift

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+
+struct ChannelLabel: View {
+  var title: String
+  var subtitle: String?
+  @Binding var isFollowing: Bool
+  
+  @ScaledMetric private var imageSize: CGFloat = 32.0
```

Comment:
I've mostly just gotten into the habit of writing out the actual types instead of relying on type inference on the rhs. I could use `Double` here though if we want

Reply:
I'd rather leave CGFloat in this case, we can leave it